src/core/network/axios/axios_operations.ts

```diff
@@ -11,8 +16,12 @@ class AxiosOperations implements IAxiosOperations {
     requestFunction: () => Promise<AxiosResponse<T>>,
   ): Promise<AxiosResponse<T>> {
     try {
+      if (store.getState().internet.isConnected) {
```

Comment:
We added the if-else condition for calling remote or Realm APIs, but its not required now, we can remove it.

Reply:
Removed.